decentraland/unity-explorer
# style: skybox improvements and refactor

## What does this PR change?

- Added sun radiance and intensity to skybox
- Added moon mask for lunar phases
- Added second sun orbiting the first sun
- Tweaked cycle colors and new values for radiance and moon mask
- Modified cloud texture to be channel coded
- (R = Value, G = Mask, B = Highlights)
- Removed dependency of second skybox texture (8k)
- Changed controller slider to use values between 0 an 1
- Changed postprocessing bloom values
- Disabled Lens Flare for all qualities (it wasnt configured)

## How to test the changes?

1. Launch de game
2. See the skybox
3. Open the debug menu and goto skybox
4. Change the value of the slider and set the time
5. Check different times of day and be amazed

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
